tap_bing_ads/__init__.py

```diff
@@ -61,6 +61,7 @@ class InvalidDateRangeEnd(Exception):
     pass
 
 def log_service_call(service_method, account_id):
+    # Log the call of service
```

Comment:
Instead of this add what this method doing

Reply:
Updated comment.